lib/bolt/project.rb

```diff
@@ -116,6 +116,7 @@ def initialize(raw_data, path, type = 'option', logs = [])
       @plans_path        = @path + 'plans'
       @managed_moduledir = @path + '.modules'
       @backup_dir        = @path + '.bolt-bak'
+      @cache_file        = @path + '.cache' + 'plugin_cache.json'
```

Comment:
```suggestion
      @cache_file        = @path + '.plugin_cache.json'
```

Since the cache is just a single file in this iteration, I think it shouldn't be saved in a separate directory, similar to .rerun.json.

Reply:
Hm, I initially made this a directory assuming we may want to cache other things in the future. I still think that might be the case, but I'm not sure what other cached elements we'll add short term (maybe commonly used modules so that using Puppet is faster?). Having this be a file for now seems fine and we can make it a directory later since users should interact with it directly.